docs/modules/android.rst

```diff
@@ -196,6 +196,7 @@ First, you need to add the following to your ProGuard rules file::
     -keepattributes LineNumberTable,SourceFile
     -dontwarn org.slf4j.**
     -dontwarn javax.**
+    -keep class io.sentry.event.Event { *; }
```

Comment:
fwiw, there may be a more granular config we could include here (and in the gradle-plugin).